src/com/company/Utils.java

```diff
@@ -0,0 +1,240 @@
+package com.company;
```

Comment:
What's this for? I don't think it's being used anywhere? Can we get rid of it?

The code rights also aren't clear - I see an external author on it, but it's doesn't have a license attached.